public/stylesheets/masters/low_vision_default/low_vision_default_site_screen_.css

```diff
@@ -142,18 +163,19 @@ pre {
   position: static;
 }
 
-#footer {
-  float: left;
-  padding: 0;
-}
-
 #header,
 #header .open a,
 #header .primary .dropdown a:focus,
 #header .user .dropdown a:focus,
 #footer,
+#footer button:hover,
+#footer button:focus,
 .actions a:hover,
-.actions a:focus {
+.actions a:focus,
+input[type="submit"]:hover,
+input[type="submit"]:focus,
```

Comment:
Can you remove `#footer button:hover, #footer button:focus` above and add `button:hover, button:focus` here, so we style all button elements, not just the ones in the footer or in actions blocks?